### PR TITLE
Upload temps and degree days from Met Office.

### PR DESCRIPTION
**Pull request by otfrom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,41 @@
 # kixi.hecuba.weather
 
-A Clojure library designed to ... well, that part is up to you.
+An additional library for kixi.hecuba to upload weather observation data.
```
Comment (otfrom) [inline README.md]: A microservice...

File: script/delete_measurements.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+DATA="/home/eleonore/Documents/weather-live-data.csv"
+
+## For each row of the file, build the url for the curl command
+for row in $(cat $DATA);
```
Comment (otfrom) [inline script/delete_measurements.sh]: `${DATA}`

File: script/run-metoffice-daily.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+java -jar kixi.hecuba.weather-0.2.0-SNAPSHOT-standalone.jar -u$khwuser -p$khwpass -c$khwdevicesfile
```
Comment (otfrom) [inline script/run-metoffice-daily.sh]: ${KHWUSER} etc...

Comment (sw1nn) [inline script/run-metoffice-daily.sh]: it's useful to do 

`${KHWUSER?NOT DEFINED}`

which will error out if the env var isn't defined.

**Pull request merged**